CHANGES.rst

```diff
@@ -198,6 +198,7 @@ New Features
   - Added furlong to imperial units. [#3529]
   - Added mil to imperial units. [#3716]
   - Added stone to imperial units. [#4192]
+  - Added Earth Mass (``Mearth``) and Jupiter mass (``M_jup``) to units [#3907]
```

Comment:
Should be ``M_earth`` - might as well make it consistent with ``M_jup`` (but include ``[ci skip]`` for this)